### PR TITLE
Merge meta into development

### PR DESCRIPTION
**Pull request by lrebscher**
No need to maintain meta and development branch anymore, further changes should happen on development directly

### CODE REVIEW

**Comment by lrebscher** [comment]
merged with `git merge -s ours development`

> This resolves any number of heads, but the resulting tree of the merge is always that of the current branch head, effectively ignoring all changes from all other branches. It is meant to be used to supersede old development history of side branches. Note that this is different from the -Xours option to the recursive merge strategy.

File: README.md [hunk 1]
```diff
@@ -2,18 +2,27 @@
   <img src="https://github.com/tarelli/bucket/blob/master/geppetto%20logo.png?raw=true" alt="Geppetto logo"/>
 </p>
 
-# Geppetto Website
+# Geppetto Meta
 
-Geppetto's Website
+Geppetto's Meta
```
Comment (filippomc) [inline README.md:7]: Geppetto MetaCell distribution

**Review by filippomc — APPROVED** [review]
Just a minor question, looks good to me.

File: geppetto-showcase/package.json [hunk 2]
```diff
@@ -14,13 +14,13 @@
     "@babel/plugin-proposal-class-properties": "^7.8.3",
     "@babel/preset-env": "^7.8.4",
     "@babel/preset-react": "^7.8.3",
-    "@geppettoengine/geppetto-client": "file:./geppetto-client",
+    "@geppettoengine/geppetto-client": "file:../geppetto.js",
     "awesome-typescript-loader": "^5.2.1",
-    "babel-eslint": "^10.1.0",
+    "babel-eslint": "^8.2.6",
```
[the remark this answers is not in the document]
Comment (lrebscher) [reply geppetto-showcase/package.json:19]: don't know, @afonsobspinto ? 😄

[the remark this answers is not in the document]
Comment (afonsobspinto) [reply geppetto-showcase/package.json:19]: I think it was unintentional. The package is deprecated and I was doing some experiments to try to add the replacement (@babel/eslint-parser) but I ended up dropping that.

**Pull request merged**